packages/amplication-client/src/Workspaces/WorkspaceFooter.scss

```diff
@@ -4,8 +4,75 @@
   display: flex;
   flex-direction: row;
   align-items: center;
+  justify-content: space-between;
   padding: 0px var(--large-spacing);
   background: var(--black10);
   height: var(--footer-height);
   min-height: var(--footer-height);
+
+  &__skeleton {
+    display: flex;
+    align-items: center;
+  }
+
+  &__vertical_border {
+    width: 1px;
```

Comment:
try to use :
width:0;
border-right:$border-black60;

Reply:
I tried. It doesn't look good.
We also do it like that in WorkspaceHeader
https://github.com/amplication/amplication/blob/master/packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.scss#L152